source/converter/contracts/Converter.sol

```diff
@@ -128,7 +128,7 @@ contract Converter is Ownable, ReentrancyGuard, TokenPaymentSplitter {
         path = getTokenPath(_swapFromToken);
       }
       // Approve token for AMM usage.
-      _approveErc20(_swapFromToken, tokenBalance);
+      IERC20(_swapFromToken).safeIncreaseAllowance(address(uniRouter), tokenBalance);
```

Comment:
``uniRouter`` is declared as address type, why is there a need to cast it to 'address' again?